python/node_classification.py

```diff
@@ -0,0 +1,342 @@
+import mgp
+from tqdm import tqdm
+from mage.node_classification.metrics import *
+from mage.node_classification.train_model import *
+from torch_geometric.data import Data
+from mage.node_classification.model.inductive_model import InductiveModel
+from mage.node_classification.utils.convert_data import convert_data
+import torch
+import numpy as np
+import sys
+import mgp
+
+##############################
+# constants
+##############################
+
+
+class ModelParams:
+    IN_CHANNELS = "in_channels"
+    OUT_CHANNELS = "out_channels"
+    HIDDEN_FEATURES_SIZE = "hidden_features_size"
+    LAYER_TYPE = "layer_type"
+    AGGREGATOR = "aggregator"
+
+
+class OptimizerParams:
+    OPTIMIZER = "optimizer"
+    LEARNING_RATE = "learning_rate"
+    WEIGHT_DECAY = "weight_decay"
+
+
+class DataParams:
+    SPLIT_RATIO = "split_ratio"
+    METRICS = "metrics"
+
+
+class MemgraphParams:
+    NODE_FEATURES_PROPERTY = "node_features_property"
+    NODE_ID_PROPERTY = "node_id_property"
+    NODE_CLASS_PROPERTY = "node_class_property"
+
+
+class TrainParams:
+    NUM_EPOCHS = "num_epochs"
+    CONSOLE_LOG_FREQ = "console_log_freq"
+    CHECKPOINT_FREQ = "checkpoint_freq"
+    TOTAL_NO_EPOCHS = "total_no_epochs"
+
+
+class OtherParams:
+    DEVICE_TYPE = "device_type"
+    PATH_TO_MODELS = "/home/mateo/mage/pytorch_models/model"
+
```

Comment:
This can't remain as the default parameter :)

Reply:
Changed to `../pytorch_models/model`.